prog/prio_test.go

```diff
@@ -59,3 +59,50 @@ func TestPrioChoice(t *testing.T) {
 		t.Fatal(diff)
 	}
 }
+
+// Test static priorities assigned based on argument direction.
+func TestStaticPriorities(t *testing.T) {
+	target, rs, _ := initTest(t)
+	iters := 100000
+	// The first call is the one that creates a resource and the rest are calls that can use that resource.
+	tests := []string{
+		`r0 = open(&(0x7f0000393000-0x8)="2e2f66696c653000", 0x10000, 0x80)
+read(r0, &(0x7f0000617000)="00", 0x1)
+write(r0, &(0x7f0000617000+0x13)="2b03c3d04417ba11", 0x8)
```

Comment:
It seems we don't really need the program and these "&(0x7f0000617000+0x13)="2b03c3d04417ba11"" don't have any meaning for the test. What we want is sequence of calls. So let's do that:
```
[]string{"open", "read", "write", "mmap"}
```
later you can turn that into Syscall with target.SyscallMap[name], or perhaps the names are enough for this test?